pkg/csi/service/vanilla/controller.go

```diff
@@ -494,6 +493,11 @@ func (c *controller) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequ
 	log.Infof("CreateVolume: called with args %+v", *req)
 
 	if common.IsFileVolumeRequest(ctx, req.GetVolumeCapabilities()) {
+		if req.Parameters[common.CSIMigrationParams] == "true" {
+			msg := "can't create file volume using VCP StorageClass"
+			log.Error(msg)
+			return nil, status.Error(codes.Internal, msg)
```

Comment:
Invalid argument instead of Internal?

Reply:
Agree with @RaunakShah

Reply:
fixed. `return nil, status.Error(codes.InvalidArgument, msg)`